packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -143,9 +144,10 @@ export default function createConnector(connectorDesc) {
         error,
         widgets,
         metadata,
+        resultsFacetValues,
       } = store.getState();
       const searchState = {results, searching, error};
-      return connectorDesc.getProvidedProps.call(this, props, widgets, searchState, metadata);
+      return connectorDesc.getProvidedProps.call(this, props, widgets, searchState, metadata, resultsFacetValues);
```

Comment:
One thing we could do for that is merging the `searchResults` object with the searchForFacetValuesResults` as soon as we get one or the other. 

One thing that bothers me though is that the `searchResults` object comes directly from the helper and in our documentation we are actually pointing to the helper documentation to explain its structure. Merging `searchResults` and `searchForFacetValuesResults` wouldn't be a little bit confusing? Can you see another ways?